src/main/java/uk/gov/laa/ccms/caab/client/CaabApiClient.java

```diff
@@ -216,6 +216,29 @@ public Mono<CostStructure> getCosts(
         .onErrorResume(caabApiClientErrorHandler::handleGetCostsError);
   }
 
+  /**
+   * Updates the cost structure of an application using the CAAB API.
+   *
+   * @param id The ID of the application.
+   * @param costs The new cost structure for the application.
+   * @param loginId The ID associated with the user login.
+   * @return A Mono Void indicating the completion of the update operation.
+   */
+  public Mono<Void> updateCosts(
```

Comment:
I know terminology varies, but maybe better to name it updateCostStructure to match the endpoint

Reply:
amended